packages/react/src/SideMenu/components/DragHandleMenu/DragHandleMenuItem.tsx

```diff
@@ -2,7 +2,7 @@ import { Menu } from "@mantine/core";
 import { PolymorphicComponentProps } from "@mantine/utils";
 
 export const DragHandleMenuItem = (
-  props: PolymorphicComponentProps<"button">
+  props: PolymorphicComponentProps<"button"> & { closeMenuOnClick?: boolean }
```

Comment:
instead of hardcoding this, can't we reuse the Mantine props? i.e.: is there a prop type of Menu.Item we can reuse?